### PR TITLE
Move test user creating from selenium tests launcher to framework

### PR DESCRIPTION
**Pull request by dmytro-ndp**
### What does this PR do?
- is moving selenium test users creation launcher to the test framework
- gets rid of second test user
- makes usage of default test user more obvious
- creates test user in time of injection TestUser instance
- creates keycloak admin console client library
- involves test user provider.

### What issues does this PR fix or reference?
#9277

### CODE REVIEW

**Comment by dmytro-ndp** [comment]
ci-test

**Comment by skabashnyuk** [comment]
@dmytro-ndp I see too many changes not related to the topic of this pr? Are you sure there is no mistake here?

**Comment by dmytro-ndp** [comment]
@skabashnyuk: don't worry - it's because of incorrect rebase from master. I will fix it before merge.

**Comment by dmytro-ndp** [comment]
ci-test

**Review by akorneta — APPROVED** [review]

**Review by musienko-maxim — APPROVED** [review]

**Review by tolusha — APPROVED** [review]

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/client/KeycloakAdminConsoleClient.java [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+  }
+
+  public TestUserImpl createUser(RemovableUserProvider testUserProvider) throws IOException {
+    if (!dockerUtil.isCheRunLocally()) {
+      throw new IOException(
+          "It's impossible to create test user because of Che is running on the different host.");
+    }
+
+    long currentTimeInMillisec = System.currentTimeMillis();
+    String username = "user" + currentTimeInMillisec;
```
Comment (Ohrimenko1988) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/client/KeycloakAdminConsoleClient.java:92]: What about using "NameGenerator" ?

**Review by SkorikSergey — APPROVED** [review]

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/client/KeycloakAdminConsoleClient.java (hunk 1 above)
Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/client/KeycloakAdminConsoleClient.java:92]: NameGenerator would be useful, but here stamp is used for the case when we will need to know test user time creation, for example, if some test user wasn't removed in time.

**Review by Ohrimenko1988 — APPROVED** [review]

**Comment by dmytro-ndp** [comment]
@akorneta: please, review fresh commit with refactoring made due to your kind suggestions.

**Comment by akorneta** [comment]
@dmytro-ndp looks good to me

**Pull request merged**